users/postgres/users.go

```diff
@@ -38,7 +38,10 @@ func NewUserRepo(db Database) users.UserRepository {
 }
 
 func (ur userRepository) Save(ctx context.Context, user users.User) (string, error) {
-	q := `INSERT INTO users (email, password, id, metadata) VALUES (:email, :password, :id, :metadata) RETURNING id`
+	if user.State == "" {
+		user.State = "active"
+	}
```

Comment:
I'd prefer not to set default here but on the service layer with the rest of the service logic.